app/views/media_objects/_metadata_display.html.erb

```diff
@@ -52,6 +52,9 @@ Unless required by applicable law or agreed to in writing, software distributed
 </dl>
 <% if can? :inspect, @media_object %>
   <%= link_to 'View Raw Metadata', inspect_media_object_path(@media_object.id, 'descMetadata'), :target => 'blank', :class => "btn btn-default btn-small" %>
+  <%# TODO 3006 move this to structure metadata page
```

Comment:
How about removing this since it is commented out and the work of wiring up this to the UI will be in a future story.